src/games/strategy/net/OpenFileUtility.java

```diff
@@ -0,0 +1,54 @@
+package games.strategy.net;
+
+import java.awt.Desktop;
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+
+import javax.swing.JOptionPane;
+
+import games.strategy.debug.ClientLogger;
+
+public class OpenFileUtility {
+
+  /**
+   * Opens a specific file on the user's computer, using whatever default program is used to open such files, using the
```

Comment:
This comment can be tightened up, you lose no information if you delete "using whatever default program is used to open such files, "

IE:  "Opens a specific file on the user's computer using the local computer's file associations."